scripts/dts/extract_dts_includes.py

```diff
@@ -215,6 +215,18 @@ def extract_property(node_compat, node_address, prop, prop_val, names):
 
     def_label = 'DT_' + def_label
 
+    if 'instance_id' in reduced[node_address]:
+        parent_address = get_parent_address(node_address)
+        if parent_address:
+            plabel = '""'
+            if 'label' in reduced[parent_address]['props']:
+                plabel = '"' + reduced[parent_address]['props']['label'] + '"'
+            instance = reduced[node_address]['instance_id']
+            for k in instance:
```

Comment:
Can iterate through key/value pairs like this:

```python
for k, i in instance.items():
    b = ...
```

Maybe a more descriptive name than `k` could be used, describing what the key represents.

Reply:
Could do this too, and get rid of the `instance` variable:

```python
for k, i in reduced[node_address]['instance_id'].items():
    b = ...
```